schemas/service.proto

```diff
@@ -778,6 +781,33 @@ message TenantIdList {
       optional bytes bookmark = 2;
 }
 
+message SearchPrincipalRequest {
```

Comment:
Please add documentation at `SearchPrincipalRequest` level

Reply:
Thank you, this shows up on our website :smile: